bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openshift-object-describer",
-  "version": "0.0.7",
```

Comment:
let's stop bumping bower versions in the same commits as improvements... I can bump them when I cut a release